recipes/pymor/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "0.5.1" %}
+
+package:
+    name: pymor
+    version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pymor/pymor-{{ version }}.tar.gz
+  sha256: ad5d0e8db09bf65580563632f5a73403790672f8ab33bc943da344fe70d290a0
+
+build:
+  number: 4
```

Comment:
why 4 and not 0 ?

Reply:
At some point I thought I had to manually bump that as counter. 

> The build number is used when the source code for the package has not changed but you need to make a new build. For example, if one of the dependencies of the package was not properly specified the first time you build a package, then when you fix the dependency and rebuild the package you should increase the build number.

Bit I guess that only refers to after the recipe has been accepted?

Reply:
yup